newsroom/agenda/utils.py

```diff
@@ -125,8 +125,10 @@ def get_coverage_status_text(coverage):
         return 'in progress at {}.'.format(get_coverage_scheduled_date(coverage))
 
     if coverage.get('workflow_status') == ASSIGNMENT_WORKFLOW_STATE.COMPLETED:
-        return '{}{}.'.format('updated' if len(coverage.get('deliveries') or []) > 1 else 'available',
-                              ' at ' + get_coverage_publish_time(coverage))
+        return '{}{}.'.format(
+            'updated' if len(coverage.get('deliveries') or []) > 1 else 'available',
```

Comment:
we should use gettext on those strings

Reply:
As this is used in templates, `gettext` won't work here unless we programatically change the gettext language for this email request.

It is for this reason I moved this into a template for FR-CA:
https://github.com/superdesk/newsroom-app-cp/blob/develop/server/theme/agenda_status_text.txt

Reply:
I see ok

Reply:
But I really dislike constructing strings like these in python for jinja templates. We should look at this at some point